android/app/src/main/java/net/minetest/minetest/GameActivity.java

```diff
@@ -203,4 +202,8 @@ public void shareFile(String path) {
 		Intent shareIntent = Intent.createChooser(intent, null);
 		startActivity(shareIntent);
 	}
+
+	public String getLanguage() {
+		return Locale.getDefault().getLanguage();
```

Comment:
The getLanguage() method still uses the old language code to preserve compatibility.

See https://www.loc.gov/standards/iso639-2/php/code_changes.php for list of code changes. We only need it for two-letter codes since we only use the two-letter language code.

```suggestion
		// The getLanguage() method still uses the old language code to preserve compatibility.
		String languageCode = Locale.getDefault().getLanguage();

		// List of code changes in ISO 639
		// See https://www.loc.gov/standards/iso639-2/php/code_changes.php
		switch (languageCode) {
			case "in":
				languageCode = "id"; // Indonesian
				break;
			case "iw":
				languageCode = "he"; // Hebrew
				break;
			case "ji":
				languageCode = "yi"; // Yiddish
				break;
			case "jw":
				languageCode = "jv"; // Javanese
				break;
		}

		return languageCode;
```